StudentList/Fragments/StudentProfileFragment.cs

```diff
@@ -136,11 +141,13 @@ public override void OnActivityResult(int requestCode, int resultCode, Intent da
                 if (data.Data == null)
                 {
                     this.profilePhoto = (Bitmap)data.GetParcelableExtra("data");
+                    this.profilePhotoGuid = Guid.NewGuid();
```

Comment:
this line and one below are same for both if-else branches, extract them outside if